src/OrchardCore.Modules/OrchardCore.Media.AmazonS3/Constants.cs

```diff
@@ -6,13 +6,12 @@ internal static class ValidationMessages
     {
         public const string BucketNameIsEmpty = "BucketName is required attribute for S3 Media";
 
-        public const string SecretKeyIsEmpty =
-            "SecretKey is required attribute for S3 Media, make sure it exists in Credentials section or ProfileName you specified";
-
-        public const string AccessKeyIdIsEmpty =
-            "AccessKeyId is required attribute for S3 Media, make sure it exists in Credentials section or ProfileName you specified";
+        public const string RegionEndpointIsEmpty = "Region is required attribute for S3 Media";
```

Comment:
I missed it last time. Shouldn't these be localized strings? Can you compare with other implementations in the repository?

Reply:
I can check

Reply:
We don't need to localize these, since these are used only to write log messages in the end in `OrchardCore.Media.AmazonS3.Startup`.